src/components/Layout/PageLayout/pageLayoutStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { Style } from "shared/types/style";
```

Comment:
import from the 'index' file that should expose all things from the 'type' folder